.github/workflows/build.yaml

```diff
@@ -200,21 +200,23 @@ jobs:
                 MERGE_METHOD: squash
                 MERGE_LABELS: ""
 
-            - name: Check for PR merge
+            - name: Get master branch sha
+              id: master_sha
               if: ${{ steps.check_version_updated.outputs.updated == 'true'}}
               run: |
-                ve1/bin/check-auto-merge --api-url=${{ github.event.pull_request._links.self.href }}
+                git fetch
+                export ORIGIN_MASTER_SHA=$(git rev-parse origin/master)
+                echo "origin_master_sha=$ORIGIN_MASTER_SHA" >> $GITHUB_OUTPUT
 
-            - name: Create the the release
-              id: create_release
+            - name: Tag master
```

Comment:
NIT: I'd probably rename this to `create release tag`.
It's also worth noting that it isn't clear what should/would happen if that tag existed already. I believe that also makes this task not re-runnable once we get to this point?

Reply:
That is correct, if the tag already exists then this step will fail.

I think we can live with this "limitation" for the following reasons:
- IMO this step should be the last step of this workflow. Currently it is followed by the "retagging" to `latest` of the quay.io container image. But ideally those steps should be moved to the "release.yaml" workflow so that all assets (tarball and container image) are created there. To be discussed of course, but if we agree this is something I would address in a separate PR to keep this one a little bit lighter.
- It's unlikely that we want to re-release a particular version. But if we need to, we can manually delete and recreate the tag. This will recreate the GitHub release (old one will become a "Draft" that needs to be manually deleted) and the tarball. Once the container image build is moved to the release workflow, they will also be recreated by this action.